<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Result</class>
 <widget class="QDialog" name="Result">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/background/background2.jpg);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">color: rgb(180, 72, 72);
font: 600 20pt &quot;URW Bookman&quot;;
</string>
     </property>
     <property name="text">
      <string>YOUR SCORE:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="score_label">
     <property name="styleSheet">
      <string notr="true">color: rgb(180, 72, 72);
font: 600 20pt &quot;URW Bookman&quot;;
</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="congrats_label">
     <property name="styleSheet">
      <string notr="true">color: rgb(180, 72, 72);
font: 600 14pt &quot;URW Bookman&quot;;</string>
     </property>
     <property name="text">
      <string>Congratulations you are on the leaderboard!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignHCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">color: rgb(180, 72, 72);
font: 600 20pt &quot;URW Bookman&quot;;
</string>
       </property>
       <property name="text">
        <string>NAME:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="result_lineEdit">
       <property name="styleSheet">
        <string notr="true">border:5px solid;
background-image: url(:/background/whiteBackground.jpg);
border-color: rgb(238, 148, 148);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="enter_result_button">
       <property name="styleSheet">
        <string notr="true">color:rgb(180, 72, 72);
font: 600 17pt &quot;URW Bookman&quot;;
border-radius:10px;
border:7px solid;
border-color: rgb(238, 148, 148);
background-image: url(:/background/backgroundBtns.jpg);</string>
       </property>
       <property name="text">
        <string>ENTER</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="menu_result_button">
     <property name="styleSheet">
      <string notr="true">color:rgb(180, 72, 72);
font: 600 17pt &quot;URW Bookman&quot;;
border-radius:10px;
border:7px solid;
border-color: rgb(238, 148, 148);
background-image: url(:/background/backgroundBtns.jpg);</string>
     </property>
     <property name="text">
      <string>MENU</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
